scan/src/rpc/TxHook.re

```diff
@@ -409,17 +501,22 @@ let atHash = txHash => {
 };
 
 let atHeight = (height, ~page=1, ~limit=25, ()) => {
-  let json = AxiosHooks.use({j|txs?tx.height=$height&page=$page&limit=$limit|j});
+  let json =
+    AxiosHooks.use({j|txs?tx.height=$height&page=$page&limit=$limit|j});
```

Comment:
Please set line length to 100 so your autoformat won't conflict with others. Ask @Benzbeeb how to do that. He did it a few weeks ago.